app/controllers/concerns/authentication.rb

```diff
@@ -23,11 +23,14 @@ def require_authentication
   end
 
   def resume_session
-    Current.session ||= find_session_by_cookie
+    session = Current.session ||= find_session_by_cookie
+    session.touch if session.present?
```

Comment:
Do we need to return the session after this or does `.touch` return the instance of the session 🤔
```suggestion
    session.touch if session.present?
    session
```

Reply:
`session.touch` returns true. Everywhere it is used (`authenticated?` and `require_authentication`) we are looking for true / false, so I think this is ok. But it is true that it returned the session in the past, so I will update